core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.html

```diff
@@ -28,6 +28,29 @@ <h4 class="modal-title">Execution results of this workflow</h4>
           ></i>
         </td>
         <td nzEllipsis>{{(i + 1)}}</td>
+        <td nzEllipsis>
+          <label *ngIf="workflowExecutionsIsEditingName.indexOf(i) === -1; else customeWorkflow " class="workflow-name"
+            >{{ row.name }}</label
+          >
+          <ng-template #customeWorkflow>
+            <input
+              #customName
+              (focusout)="confirmUpdateWorkflowExecutionsCustomName(row, customName.value, i)"
+              (keyup.enter)="confirmUpdateWorkflowExecutionsCustomName(row, customName.value, i)"
```

Comment:
seems if the user clicks on the pencil and decides to click somewhere out without entering, then there is no way of exiting the edit mode

Reply:
default execution name variable has been removed